### ISSUE TITLE
Training-Loss error does not decrease no matter what I change

### ISSUE THREAD

**Issue by rdanesh1367** [anishmadan23/deep3d-pytorch#1]
I want to train your model on my data (stereo data from surgery) but the error is not changing. I also tried to train it on 1 sample but the error sweeps from .0002 to .00015 and does not go lower (It is changing but not decreasing). Do you have any idea what can I do to make it learn? I changed the learning rate but does not change anything.

**Comment by rdanesh1367** [comment]
I found the answer. the problem was data initialization that divided each image by 255 and it maid the error so small that it vanished in bach-propagation and the net could not be trained.
